src/coordsim/simulation/simulatorparams.py

```diff
@@ -6,12 +6,11 @@
 other parameters for the simulator.
 
 """
+import numpy as np
 
 
 class SimulatorParams:
-    def __init__(self, network, ing_nodes, sfc_list, sf_list, config, seed, schedule={}, sf_placement={}):
-        # Seed for the random generator: int
-        self.seed = seed
+    def __init__(self, network, ing_nodes, sfc_list, sf_list, config, schedule={}, sf_placement={}):
```

Comment:
Apparently, default `{}` for optional args can really lead to unintended behavior. If it's optional, rather default to `None` and init with empty dict inside the function. See https://stackoverflow.com/a/26320938/2745116